src/charm.py

```diff
@@ -145,32 +137,32 @@ def _configure_nrf(self, event: EventBase) -> None:
             return
         if not self._get_database_uri():
             self.unit.status = WaitingStatus("Waiting for database URI")
-            event.defer()
             return
         if not self._container.exists(path=BASE_CONFIG_PATH):
```

Comment:
it just came to my attention that we verify that `/etc/nrf/` is mounted but we do not verify `/support/TLS`
should we check both ?

Reply:
good point, yes, I would think so

Reply:
done